### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD036.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#39] [General]
# https://blog.3place.cn/3place/fsd/FSD036.html

https://blog.3place.cn/3place/fsd/FSD036.html

**Comment by picklecai** [comment]
我的眼睛被引向汪洋大海；
看那海水高涨，堆叠如山，
随即后退，翻滚着波涛澎湃，
袭击一片辽阔而平坦的海岸。

在浮士德眼里，涨潮或者海啸，原来是暴力欺负手无寸铁的平民。

**Comment by picklecai** [comment]
梅菲斯特给了浮士德一个取得声名和基地的机会。
